gflownet/envs/crystals/composition.py

```diff
@@ -165,47 +170,76 @@ def get_mask_invalid_actions_forward(self, state=None, done=None):
         n_unused_required_elements = len(unused_required_elements)
         n_used_atoms = sum(state)
 
+        if self.do_spacegroup_check and isinstance(self.space_group, int):
+            space_group = get_space_group(self.space_group)
+
+            # Determine, based on the space group's Wyckoff's positions, what
+            # is the min/max number of atoms of a given element that could be
+            # added.
+            most_specific_wp = space_group.get_wyckoff_position(-1)
+            min_atom_i = most_specific_wp.multiplicity
+            max_atom_i = (self.max_atom_i // min_atom_i) * min_atom_i
```

Comment:
This may potentially increase the minimum number of atoms and decrease the maximum number of atoms with respect to the config values, right?

Reply:
Btw, thanks for the comment. It made me revisit the code and find a bug.
I have originally written this making the assumption that, for a space group, all WPs are multiples of the most specific WP. 
I figured I would validate that assumption, and fix the code if needed, once the pipeline was working. Well, I forgot to do that before doing my PR.
Your comment made me revisit that bit of code and fix it because I checked and there are 4 space groups for which my assumption was invalid.

Reply:
Well, glad to hear :) And thanks for the explanation!